src/transformers/modeling_tf_transfo_xl.py

```diff
@@ -745,13 +764,24 @@ def __init__(self, config):
         self.sample_softmax = config.sample_softmax
         # use sampled softmax
         if config.sample_softmax > 0:
-            raise NotImplementedError
+            self.out_layer = TFTransfoXLLMHead(config, self.transformer.word_emb.weight, name="out_layer")
```

Comment:
As is the case in PT: `self.transformer.word_emb.weight` does not work here, but added it for consistency.